browser/ui/browser_commands.cc

```diff
@@ -410,4 +411,112 @@ void CloseDuplicateTabs(Browser* browser) {
   }
 }
 
+bool CanCloseTabsToLeft(Browser* browser) {
```

Comment:
As we reimplemented query functions like `CanXXX()` instead of using upstream,
it would be good to have tests for these functions if it's not covered yet.

Reply:
Hey, do you have any pointers to something else testing mute functionality?

Reply:
Well.. it looks like `content/browser/media/session/media_session_controller_unittest.cc` uses mute api.
but, let's skip test for mute api as it just call web contents api w/o additional logic.
IMO, adding tests for `CloseTabs...` and `Unpin..` seems sufficient.